hail/python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -856,40 +830,34 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def plDosage(pl):\n",
-    "    linear_scaled = pl.map(lambda x: 10 ** (-x / 10))\n",
-    "    pl_sum = hl.sum(linear_scaled)\n",
-    "    normed = linear_scaled / pl_sum\n",
-    "    return 1 * normed[1] + 2 * normed[2]"
+    "p = hl.plot.qq(linear_regression_results.p_value)\n",
+    "show(p)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "linear_regression_results = hl.linear_regression_rows(\n",
-    "    y=cmt.pheno.CaffeineConsumption, \n",
-    "    x=plDosage(cmt.PL),\n",
-    "    covariates=[1.0, cmt.pheno.isFemale, cmt.pca.scores[0], cmt.pca.scores[1], cmt.pca.scores[2]])"
+    "That's more like it! This shape is indicative of a well-controlled (but not especially well-powered) study. And now for the Manhattan plot:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
-    "p = hl.plot.qq(linear_regression_results.p_value)\n",
+    "p = hl.plot.manhattan(linear_regression_results.p_value)\n",
     "show(p)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "That's more like it! We may not be publishing ten new coffee-drinking loci in _Nature_, but we shouldn't expect to find anything but the strongest signals from a dataset of 284 individuals anyway. "
+    "We have found a caffeine consumption locus! Off to publish in _Nature_!"
```

Comment:
"Now simply apply Hail's Nature paper function to publish the result. Just kidding, that function won't land until Hail 1.0!"